app/models/comment.rb

```diff
@@ -303,6 +307,16 @@ def published_article
     errors.add(:commentable_id, "is not valid.") if commentable_type == "Article" && !commentable.published
   end
 
+  def user_mentions_in_markdown
+    return if created_at.present? && (created_at < MAX_USER_MENTION_LIVE_AT)
```

Comment:
Just a random suggestion, but `ActiveSupport` adds some nice helpers that make dealing with timestamps a bit more explicit and easier to follow:

```suggestion
   return if created_at.present? &&  created_at.before?(MAX_USER_MENTION_LIVE_AT)
```